model/qti/ImportService.php

```diff
@@ -548,13 +551,19 @@ public function importQtiItem(
                 $itemAssetManager->loadAssetHandler($peHandler);
 
                 if ($this->getServiceLocator()->get(\common_ext_ExtensionsManager::SERVICE_ID)->isInstalled('taoMediaManager')) {
+                    // Media manager path where to store the stimulus.
+                    $path = $this->retrieveFullPathLabels($itemClass);
```

Comment:
If we reflect a path of an item on import, shouldn't we do the same on a regular media file upload during authoring?

Reply:
That's a good question. The specificity of shared stimulus imported from a package where it's used is that the shared stimulus is directly linked to a set of items. So reflecting the path makes sense. When manually uploading an asset, it is not specifically linked to an item, so there's no way to know which tree structure to replicate.